raddb/sites-available/default

```diff
@@ -749,7 +749,7 @@ recv Access-Request {
 	#  filter_username policy.
 	#
 	filter_username
-
+	imap
```

Comment:
it shouldn't be enabled here in the default config.  and it probably should only be in the `authenticate` section.

Reply:
Agreed, shouldn't be in the default config, but does need to be in authorize and authenticate, for the same reason as pap et al...